.circleci/config.yml

```diff
@@ -40,20 +40,24 @@ workflows:
   build:
     jobs:
       - "openjdk8-scala2.12":
-          filters: # required since `openjdk8-scala2.12.8_deploy` has tag filters AND requires `openjdk8-scala2.12.8`
+          # required since openjdk8-scala2.12_deploy has tag filters AND requires
+          # openjdk8-scala2.12
+          # https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag
+          filters:
             tags:
               only:
                 - /^(.*)$/
       - "openjdk8-scala2.12_deploy":
+          context: sonatype
```

Comment:
>Contexts provide a mechanism for securing and sharing environment variables across projects. The environment variables are defined as name/value pairs and are injected at runtime.

This will allow us to reuse the signing key across other projects in the Azavea org without the requirement for Ops to get involved.

See: https://circleci.com/docs/2.0/contexts/

Reply:
My instincts to name this something more specific are kicking in a bit. I realize that we've got different signing keys, and that right now those signing keys roughly match up to different CircleCI orgs (so each org could have a `sonatype` context and it would work), but I could see a situation where they need to mix or we need to add an additional signing key/Sonatype credential combination within an org.

I think it is worth considering a name that more closely maps to the values of the variables encapsulated by this context. For example, `sonatype-azavea-signing-key`, or something similar.

Reply:
>[...] but I could see a situation where they need to mix or we need to add an additional signing key/Sonatype credential combination within an org.

>I think it is worth considering a name that more closely maps to the values of the variables encapsulated by this context.

Agreed. Thanks for taking a pass at this.